### PR TITLE
fix(docs): update technical accuracy

### PR DESCRIPTION
**Pull request by akoumpa**
# What does this PR do ?

Add a one line overview of what this PR aims to accomplish.

# Changelog

- Add specific line by line info of high level changes in this PR.

# Before your PR is "Ready for review"

**Pre checks**:

- [ ] Make sure you read and followed [Contributor guidelines](https://github.com/NVIDIA-NeMo/Automodel/blob/main/CONTRIBUTING.md)
- [ ] Did you write any new necessary tests?
- [ ] Did you add or update any necessary documentation?

If you haven't finished some of the above items you can still open "Draft" PR.

# Additional Information

- Related to # (issue)

### CODE REVIEW

File: .agents/contributor-skills/build-and-dependency/SKILL.md [hunk 1]
```diff
@@ -120,31 +126,20 @@ export HF_HOME="/path/to/hf_cache" # Hugging Face cache directory
...
+# The YAML's recipe field selects LLM, VLM, diffusion, or retrieval behavior.
+automodel examples/llm_finetune/llama3_2/llama3_2_1b_squad.yaml --nproc-per-node 8
 ```
 
 Override any config value from the CLI:
 
 ```bash
-automodel finetune llm -c config.yaml --model.name_or_path meta-llama/Llama-3.2-1B
+automodel examples/llm_finetune/llama3_2/llama3_2_1b_squad.yaml \
+    --model.pretrained_model_name_or_path meta-llama/Llama-3.2-1B
```
Comment (akoumpa) [inline .agents/contributor-skills/build-and-dependency/SKILL.md:142]: ok

File: docs/about/key-features.mdx [hunk 2]
```diff
@@ -76,7 +76,7 @@ FP8 training via torchao for reduced memory and higher throughput on supported m
 </Card>
 
 <Card title="Multi-Node with SLURM">
-Add a `slurm:` section to any YAML config and launch with the `automodel` CLI. See the [Cluster guide](/job-launchers/slurm-cluster).
+Copy the checked-in `slurm.sub` or download it from Github (available [here](https://github.com/NVIDIA-NeMo/Automodel/blob/main/slurm.sub)), edit its `CONFIG` and `#SBATCH` directives for your cluster, and submit it with `sbatch`. See the [Cluster guide](/job-launchers/slurm-cluster).
```
Comment (akoumpa) [inline docs/about/key-features.mdx:79]: I've added the slurm.sub link

File: docs/guides/diffusion/finetune.mdx [hunk 3]
```diff
@@ -15,14 +15,14 @@ Under the hood, NeMo AutoModel uses [flow matching](https://arxiv.org/abs/2210.0
 ┌──────────────┐    ┌──────────────┐    ┌──────────────┐    ┌──────────────┐    ┌──────────────┐
 │ 1. Install   │--->│ 2. Prepare   │--->│ 3. Configure │--->│  4. Train    │--->│ 5. Generate  │
 │              │    │    Data      │    │              │    │              │    │              │
-│ pip install  │    │ Encode to    │    │ YAML recipe  │    │ torchrun     │    │ Run inference│
+│ uv sync      │    │ Encode to    │    │ YAML recipe  │    │ torchrun     │    │ Run inference│
```
Comment (akoumpa) [inline docs/guides/diffusion/finetune.mdx:18]: switching to uv to avoid user-issues; currently, pip requires additional work to be supported.

File: docs/guides/llm/column-mapped-text-instruction-dataset.mdx [hunk 4]
```diff
@@ -235,18 +235,23 @@ full examples (Pyxis, bare-metal, Apptainer).
 ### Multi-Node Slurm Configuration
 
 <Note>
-**Multi-Node Training**: When using Hugging Face datasets in multi-node setups, you need shared storage accessible by all nodes. Set `HF_DATASETS_CACHE` to a shared directory in your sbatch script (e.g., `export HF_DATASETS_CACHE=/shared/hf_cache`) to ensure all nodes can access the cached datasets.
+**Multi-Node Training**: A shared Hugging Face cache is optional. Using a shared
```
Comment (akoumpa) [inline docs/guides/llm/column-mapped-text-instruction-dataset.mdx:238]:
```suggestion
**Multi-Node Training**: A shared Hugging Face cache is optional, but recommended. Using a shared
```

File: docs/guides/llm/dataset.mdx [hunk 5]
```diff
@@ -170,12 +170,15 @@ To enable packed sequences, add these keys to your recipe's YAML config:
 packed_sequence:
    # Set packed_sequence_size > 0 to run with packed sequences
    packed_sequence_size: 1024
-   split_across_pack: False
+   packing_strategy: thd
```
Comment (akoumpa) [inline docs/guides/llm/dataset.mdx:173]: `packing_strategy: {neat,thd}`; non-redundant arg.

File: docs/guides/llm/pretraining.mdx [hunk 6]
```diff
@@ -325,422 +322,13 @@ In these scenarios, you can pass `is_meta_device: true` in the model config. The
...
 ```bash
-uv run torchrun --nproc-per-node=2 examples/llm_pretrain/pretrain.py --config examples/llm_pretrain/megatron_pretrain_gpt2.yaml
+uv run automodel examples/llm_pretrain/megatron_pretrain_gpt2.yaml --nproc-per-node 2
 ```
 
 ### Sample Output
 
 You should see step‑wise logs reporting loss, memory usage, and tokens per second. Checkpoints will be saved under the `checkpoints/` directory as configured.
 
-```bash
```
Comment (akoumpa) [inline docs/guides/llm/pretraining.mdx:335]: removing log

File: docs/guides/llm/pretraining.mdx [hunk 7]
```diff
@@ -199,9 +200,11 @@ Below is the configuration from `examples/llm_pretrain/megatron_pretrain_gpt2.ya
 # limitations under the License.
 
 # To run this recipe, please use the following command:
-# torchrun --nproc-per-node=8 examples/llm_pretrain/pretrain.py --config examples/llm_pretrain/megatron_pretrain_gpt2.yaml
+# automodel examples/llm_pretrain/megatron_pretrain_gpt2.yaml --nproc-per-node 8
```
Comment (akoumpa) [inline docs/guides/llm/pretraining.mdx]:
```suggestion
# uv run automodel examples/llm_pretrain/megatron_pretrain_gpt2.yaml --nproc-per-node 8
```

File: docs/guides/omni/gemma3-3n.mdx [hunk 8]
```diff
@@ -209,17 +194,16 @@ To run PEFT with Gemma 3:
...
-The LoRA configuration excludes vision and audio components from adaptation to preserve pre-trained visual representations:
+The LoRA configuration excludes vision components and the language-model head from adaptation to preserve pre-trained visual representations:
 
 ```yaml
 peft:
-  peft_fn: nemo_automodel._peft.lora.apply_lora_to_linear_modules
+  _target_: nemo_automodel.components._peft.lora.PeftConfig
   match_all_linear: False
-  exclude_modules:  # exclude all vision and audio modules and lm_head
+  exclude_modules:  # exclude all vision modules and lm_head
```
Comment (akoumpa) [inline docs/guides/omni/gemma3-3n.mdx]:
```suggestion
  exclude_modules:  # exclude vision, audio and lm_head modules
```

File: docs/guides/omni/gemma3-3n.mdx [hunk 9]
```diff
@@ -209,17 +194,16 @@ To run PEFT with Gemma 3:
...
     - "*vision_tower*"
     - "*vision*"
     - "*visual*"
```
Comment (akoumpa) [inline docs/guides/omni/gemma3-3n.mdx:206]:
```suggestion
    - "*visual*"
    - "*audio*"
```

File: docs/guides/omni/gemma3-3n.mdx [hunk 10]
```diff
@@ -287,21 +272,20 @@ For models trained on MedPix-VQA, load the trained checkpoint and generate outpu
...
+
 Run the following command to load and generate from adapters trained on MedPix-VQA:
 
 ```bash
 uv run examples/vlm_generate/generate.py \
     --checkpoint-path peft_vlm_checkpoints/epoch_0_step_200/ \
     --prompt "What medical condition is shown in this image?" \
     --image-url medical_image.jpg \
-    --base-model google/gemma-3-4b-it \
-    --is-peft \
```
Comment (akoumpa) [inline docs/guides/omni/gemma3-3n.mdx:303]: confirmed there's no peft

File: docs/guides/speculative/eagle.mdx [hunk 11]
```diff
@@ -567,7 +557,7 @@ Expected output:
 
 EAGLE-1 is simpler and faster to train, making it a good starting point for
 experimentation. It uses a single transformer layer and trains with a combined
-hidden-state MSE + token cross-entropy loss.
+hidden-state SmoothL1 (Huber) + token cross-entropy loss.
```
Comment (akoumpa) [inline docs/guides/speculative/eagle.mdx:560]: nemo_automodel/components/speculative/eagle/core_v12.py:68 sets self.hidden_loss_fn = nn.SmoothL1Loss(reduction="none").

**Review by HuiyingLi — APPROVED** [review]
